src/Mayaqua/Network.c

```diff
@@ -663,17 +663,15 @@ LIST *GetNicList()
 
 #ifdef	OS_WIN32
 	o = Win32GetNicList();
-#endif	// OS_WIN32
 
-#ifdef	UNIX_LINUX
-#endif	// UNIX_LINUX
+        if (o != NULL)
+        {
+                return o;
+        }

```

Comment:
Wrong indentation (spaces instead of tabs).

Reply:
Are there coding rules defined anywhere for SE?

Reply:
No, I think that we should write them in the Wiki.

Reply:
nobody reads wiki.
if you really care about tabs and space (but from any regular developer it makes no sense, we do not use python), please find a time to add a check like that https://github.com/openresty/lua-nginx-module/blob/master/.travis.yml#L62-L63


otherwise, it is not "wrong"

Reply:
The correct term would be "inconsistent".

We can use `clang-tidy` to check that the code style is consistent.

Reply:
There was a nice code convention format I saw couple of weeks ago, where you define one set of rules and they are understood by most IDE's. Need to search for it

Reply:
Found it:
https://editorconfig.org/#file-format-details

This one offers a universal format and could be used through multiple IDE's

Reply:
it is your private opinion, no rules are defined for SE

Reply:
OK guys. Let's just define an editconfig file. Ideally using either gnu or linux kernel format or so

Reply:
I like the current code style, except for the brackets position.

I would prefer
```c
if (condition) {
	Debug("condition == true");
}
```
instead of
```c
if (condition)
{
	Debug("condition == true");
}
```

What do you think?

Reply:
guys, please start another issue discussing tabs, spaces, whatever.

this particular PR was open when no rules were defined, it does not break any rule

Reply:
It's true that no rules are defined, but it's bad to have different code styles.

Reply:
I do not think so.

Reply:
what is the difference?

Reply:
The margin.

Reply:
it is C, not python.

Reply:
That doesn't change the fact that the spacing looks different, it's not about the language.

Reply:
I understand what you are trying to say. Rules exists in your own head, but some people ask you to define rules first before you can use it. Right ?

Reply:
for C language both are equivalent.

Reply:
I understand that we don't have any rules now, but the current code style is consistent and it probably has been for the past 15 years. Why do you want to break it just because we don't have any rules which "enforce" a well-defined code style?

Reply:
I'm not trying to break anything. I simply used x windows buffer, copied few lines and pasted them. Probably, tabs were converted to spaces. I do not think it is important, and I was not doing that in purpose. For C language those things are equivalent. If you want rules, ok, but define them first.

Reply:
I'm not blaming you, I didn't say you did that on purpose.

But still, we should replace the spaces with tabs before merging.

Reply:
I do think we should. Please replace if you wish so.

Reply:
Done.

Reply:
thank you.

Reply:
You're welcome.